src/services/schema-sender.js

```diff
@@ -0,0 +1,35 @@
+const P = require('bluebird');
+const agent = require('superagent-promise')(require('superagent'), P);
+const config = require('../config');
+
+function SchemaSender(serializedSchema, secret, logError) {
+  this.perform = () =>
+    agent
+      .post(`${config.serverHost}/forest/apimaps`)
+      .set('forest-secret-key', secret)
+      .send(serializedSchema)
```

Comment:
The schema serialized here does not seem to be exactly the same as the one sent on server start.
The diff looks the same but the `apimapHash` is not the same. As a consequence the server will start working for nothing (to process the "new" apimap).
It must be fixed.